doc/integrator/headers.rst

```diff
@@ -0,0 +1,26 @@
+.. _integrator_headers:
+
+Headers
+=======
+
+We can set with some configuration some HTTP headers on defferent path.
+We do that with the ``global _headers`` from the configuration with the following syntax:
+
+.. code:: yaml
+
+    vars:
+      global_headers:
+        - pattern: <regex>
+          headers:
+            <header>: <value>
+
+If a path match more than one pattern, all the headers will be applyed, if one header applyed tweice
```

Comment:
Suggestion:
If a path matches more than one pattern, all headers listed in each match will be applied. If the same header is matched more than once, the last value is kept.